app/controllers/api/v1/tokens_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class TokensController < Doorkeeper::TokensController
```

Comment:
Does this file(`tokens_controller.rb`) required?

Reply:
I'm going to override the function `revocation_error_response ` from `Doorkeeper::TokensController` to follow `jsonapi.org`, so I think we can keep this fire here